src/panels/lovelace/editor/config-elements/hui-logbook-card-editor.ts

```diff
@@ -77,25 +93,19 @@ export class HuiLogbookCardEditor
         .computeLabel=${this._computeLabelCallback}
         @value-changed=${this._valueChanged}
       ></ha-form>
-      <h3>
-        ${`${this.hass!.localize(
-          "ui.panel.lovelace.editor.card.generic.entities"
-        )} (${this.hass!.localize(
-          "ui.panel.lovelace.editor.card.config.required"
-        )})`}
-      </h3>
-      <ha-entities-picker
+
+      <ha-target-picker
         .hass=${this.hass}
-        .value=${this._entities}
         .entityFilter=${filterLogbookCompatibleEntities}
+        .value=${this._targetPicker}
+        addOnTop
```

Comment:
```suggestion
        add-on-top
```

sorry the attribute is `add-on-top`